pennylane/gradients/general_shift_rules.py

```diff
@@ -20,22 +20,25 @@
 import pennylane as qml
 
 
-def _process_shifts(rule, tol=1e-10):
+def _process_shifts(rule, tol=1e-10, no_duplicates=False):

```

Comment:
Currently, this happens when we call `process_shifts` where previously we called `parameter_shift.py::_process_grad_recipe`.
At least this is equivalent, because `_process_grad_recipe` did not perform the check for duplicate shifts :)

Reply:
Got it!